src/main/webapp/app/exercises/programming/assess/code-editor-tutor-assessment-inline-feedback.component.ts

```diff
@@ -128,11 +128,10 @@ export class CodeEditorTutorAssessmentInlineFeedbackComponent {
             if (feedback.detailText) {
                 feedbackText = feedbackText + '\n' + feedback.detailText;
             }
-            return convertToHtmlLinebreaks(feedbackText);
+        } else if (feedback.detailText) {
+            feedbackText = feedback.detailText;
         }
-        if (feedback.detailText) {
-            return feedback.detailText;
-        }
-        return feedbackText;
+
+        return convertToHtmlLinebreaks(feedbackText);
     }
```

Comment:
This method is quite similar to the one in `feedback.model.ts`. Do you see an uncomplicated way to refactor the logic in order to implement it once and reuse it at the two spots?

Reply:
Yes, after I had another look it was quite simple. I implemented it in a281e067aaa725fdc4c55a922d0e4325eb630f08.